presidio-analyzer/pyproject.toml

```diff
@@ -34,10 +34,15 @@ stanza = { version = "*", optional = true }
 spacy_stanza = { version = "*", optional = true }
 azure-ai-textanalytics = { version = "*", optional = true }
 azure-core = { version = "*", optional = true }
+transformers = { version = "*", optional = true }
+huggingface_hub = { version = "*", optional = true }
 
 [tool.poetry.extras]
 server = ["flask"]
-transformers = ["spacy_huggingface_pipelines"]
+transformers = [
```

Comment:
@SharonHart, these need to be added right?

Reply:
Yes